docs/blog/newsletter-february-2020.rst

```diff
@@ -0,0 +1,96 @@
+Supporting documentation, documenting support
+---------------------------------------------
+
+A recent question about how to balance docs work and support content produced some interesting responses. The original post assumed two distinct teams, one for documentation and one for content produced by a support team, with each team producing distinct content. This is evidently the situation in many companies. For example, one poster described a workflow where answers to frequently-asked support questions are added to the documentation, but if a potential docs page seems particularly complicated, or addresses an edge case, the writer asks support to document it. This story describes distinct teams -- Docs and Support -- and distinct types of content.
+
+But many replies to the original post were from people who work on support teams, not on dedicated docs teams. And they write the docs. In their situations, there isn't a clear distinction between roles, nor between types of content. More than one reply described scenarios in which all docs are produced by support. 
+
+Still other replies came from technical writers whose role emerged from support. Some writers report to a support manager, and some people in a dedicated writer role work from support tickets, to make sure the docs address known customer issues. Some folks work on both product docs and separate knowledgebase articles.
+
+In short, there are myriad ways to organize the relationship between docs teams and support teams, and between the different (or not-so-different) types of content. What everyone seemed to agree on, though, was the need for close and well-coordinated communication across teams and deliverables.
+
+--------------------------------------
+How to set priorities in documentation
+--------------------------------------
+
+So you're no longer the only documentarian in your company. Hooray! Working in a team allows you to tackle bigger and more complicated projects. But how do you decide what to focus on, especially for new team members? This month, our community discussed what helps them set priorities:
+
+* Discuss the project's roadmap with project/product managers, and ask them (and other stakeholders) about immediate documentation needs. For your new writers, ask about topics that could use a little extra attention - an opportunity for them to specialize in something.
+* Encourage new writers to sit in during backlog grooming and project demos. Product prioritization can inform your own priorities.
+* Coordinate with engineering teams by attending their standup and planning meetings. You're working towards a common goal, so it helps to know how everyone's contributions fit together.
+* Try an issue tracking tool (eg JIRA or GitHub Issues). Some feel it helps keep them accountable and prevents them from struggling under the workload. Similarly, for big projects, try mapping them out using a tool, to give your team and managers a high-level glance at everyone's workload.
```

Comment:
s/JIRA/Jira (how the product itself capitalizes, although it's true it's not consistent everywhere. But I believe it isn't an acronym, so might as well keep to the main UI as a reference point)

Reply:
thanks for spotting, fixed!